influxdb_connector/influxdb_write_tester.py

```diff
@@ -34,44 +58,40 @@
 num_col = len(columns)
 for idx, row in df.iterrows():
     point = (
-    Point("ts") \
-    .time(row[columns[0]]) \
-    .field(columns[1], row[columns[1]]) \
-    .field(columns[2], row[columns[2]]) \
-    .field(columns[3], row[columns[3]]) \
-    .field(columns[4], row[columns[4]]) \
-    .field(columns[5], row[columns[5]]) \
-    .field(columns[6], row[columns[6]]) \
-    .field(columns[7], row[columns[7]]) \
-    .field(columns[8], row[columns[8]]) 
+        Point("ts")
+        .time(row[columns[0]])
+        .field(columns[1], row[columns[1]])
+        .field(columns[2], row[columns[2]])
+        .field(columns[3], row[columns[3]])
+        .field(columns[4], row[columns[4]])
+        .field(columns[5], row[columns[5]])
+        .field(columns[6], row[columns[6]])
+        .field(columns[7], row[columns[7]])
+        .field(columns[8], row[columns[8]])
     )
-    print('writing row: ', idx)
+    print("writing row: ", idx)
     write_api.write(bucket=bucket, org="kris.pan@cnvrg.io", record=point)
     time.sleep(0.01)
-
 query_api = client.query_api()
 query = 'from(bucket: "' + bucket + '")\n |> range(start: -10y)'
 tables = query_api.query(query, org=org)
 csv_builder = dict()
-csv_builder['time'] = []
+csv_builder["time"] = []
 
 for table in tables:
     for record in table.records:
-        if record['_field'] not in csv_builder:
-            csv_builder[record['_field']] = []
-        csv_builder[record['_field']].append(record['_value'])
```

Comment:
need comments to clarify logical steps

Reply:
influxdb_write_tester.py replaced with influxdb_write_tester.ipynb to support direct csv writing and better interactivity